apps/aecore/lib/aecore/structures/block.ex

```diff
@@ -5,15 +5,15 @@ defmodule Aecore.Structures.Block do
   alias Aecore.Structures.Block
   alias Aecore.Structures.Header
 
-  @type block :: %Block{}
+  @type t :: %Block{}
```

Comment:
**Note:** that there is no _correct_ way to typespec elixir structs: `%Block{}` is as acceptable as `Block.t`, its just a matter of team preference and what gets the job done. I can easily revert this change if desired.

:question: Which style does the team prefer; `%Block{}` or `Block.t`?

Reply:
%Block{} looks better imo